.github/workflows/release-napi.yml

```diff
@@ -0,0 +1,183 @@
+name: Release NAPI
+
+permissions: {}
+
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - napi/package.json # make new release on version change of napi package
```

Comment:
Start with being able to release locally.
Before adding automation, I dont think we would want every merge to main to become a release.

Reply:
Again see my main comment.
I don't think we should release platform specific versions _yet_.
WASM is universal.